src/client/js/client/gmeNodeSetter.js

```diff
@@ -28,32 +28,56 @@ define([], function () {
         }
 
         function setAttributes(path, name, value, msg) {
+            var error;
+
             if (state.core && state.nodes[path] && typeof state.nodes[path].node === 'object') {
-                state.core.setAttribute(state.nodes[path].node, name, value);
+                error = state.core.setAttribute(state.nodes[path].node, name, value);
+                if (error instanceof Error) {
```

Comment:
Yes. Basically the error only means, that although the type of the parameters are probably right, the circumstances do not allow the execution of the function.